openapi/components/schemas/Tracking/TaxTracking/TaxJarTracking.yaml

```
@@ -0,0 +1,61 @@
+type: object
+title: TaxJar
+readOnly: true
+properties:
+  id:
+    allOf:
+      - $ref: ../../ResourceId.yaml
```

```suggestion
    $ref: ../../ResourceId.yaml
```
You need to use `allOf` if you were to change the description or something like that.